app/controller.go

```diff
@@ -480,18 +461,35 @@ func handleBlockRewards(validators *identity.ValidatorStore, rewards *rewards.Re
 			continue
```

Comment:
If the Validator votes in the last block , and is removed from the validator list in the present block , would he still get the rewards for his last vote ?

Reply:
Yea, technically the validator participated in creating the block so it will get the reward for that one.

Reply:
The staking updates in the blockender of the previous block, I assumed that the present block would have the updated validator list.And Get would iterate over that. 
But as long as it's tested, I am fine with it